Lib/asyncio/sslproto.py

```diff
@@ -679,12 +692,18 @@ def _fatal_error(self, exc, message='Fatal error on transport'):
             })
         if self._transport:
             self._transport._force_close(exc)
+            self._transport = None
```

Comment:
added cleanup code for `_shutdown_timeout_handle`, it is possible to get here while handle still active